cg/cli/post_process/post_process.py

```diff
@@ -35,4 +35,19 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
-post_process_group.add_command(post_process_sequencing_run)
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.pass_obj
+def post_process_all_runs(context: CGConfig, dry_run: bool) -> None:
+    """Post-process all runs from the instruments."""
+    services: PostProcessingServices = context.post_processing_services
+    are_all_services_successful: bool = True
+    for service in [services.pacbio]:
+        is_service_successful: bool = service.post_process_all(dry_run=dry_run)
+        are_all_services_successful = are_all_services_successful and is_service_successful
+    if not are_all_services_successful:
+        raise click.Abort
```

Comment:
What does this do exactly? We are already on the last line of the command so aborting should not do anything right?

Reply:
It finishes with status != 0 and prints that in stdout. I have not stopped to question this before, maybe is not needed but I will leave it for now

Reply:
But are these not the patterns we want to break when writing new code?

Reply:
Is it? This is how the Illumina post-processing works now and I don't think there is any complain about it. Why is this pattern undesirable?

Reply:
Resolved IRL